js/signin.js

```diff
@@ -1,114 +1,82 @@
-const QUERY_SELECTOR = function (select) {
-  return document.querySelector(select);
-};
+import { $, SIGN_INFO_REGEX } from './common.js';
+import {
+  setEmailErrorStyle,
+  resetEmailStyle,
+  setPasswordErrorStyle,
+  resetPasswordStyle,
+  styleLoginError,
+} from './signErrorStyle.js';
+import { hidePassword, showPassword } from './passwordShowStyle.js';
 
-const $emailInput = QUERY_SELECTOR('#emailInput');
-const $emptyEmailError = QUERY_SELECTOR('#emptyEmailError');
-const $wrongEmailError = QUERY_SELECTOR('#wrongEmailError');
+const $jsEmailInput = $('.js-email-input');
+const $jsPasswordInput = $('.js-password-input');
 
-$emailInput.addEventListener('focusout', function (e) {
-  const emailRegEx = /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-Za-z0-9\-]+/;
+$jsEmailInput.addEventListener('blur', (e) => {
+  resetEmailStyle();
 
-  defaultStyle();
+  if (e.target.value === '') {
+    setEmailErrorStyle('이메일을 입력해주세요.');
+    return;
+  }
 
-  if (this.value === '') {
-    styleSeting('email', 'empty');
-  } else if (!emailRegEx.test(this.value)) {
-    styleSeting('email', 'wrong');
+  if (!SIGN_INFO_REGEX.email.test(e.target.value)) {
+    setEmailErrorStyle('올바른 이메일 주소가 아닙니다.');
+    return;
   }
 });
 
-const $passwordInput = QUERY_SELECTOR('#passwordInput');
-const $emptyPasswordError = QUERY_SELECTOR('#emptyPasswordError');
+$jsPasswordInput.addEventListener('blur', (e) => {
+  resetPasswordStyle();
 
-$passwordInput.addEventListener('focusout', function (e) {
-  defaultStyle();
+  if (e.target.value === '') {
+    setPasswordErrorStyle('비밀번호를 입력해주세요.');
+    return;
+  }
 
-  if (this.value === '') {
-    styleSeting('password');
+  if (!SIGN_INFO_REGEX.password.test(e.target.value)) {
+    setPasswordErrorStyle('비밀번호는 영문, 숫자 조합 8자 이상 입력해 주세요.');
   }
 });
 
-const $loginButton = QUERY_SELECTOR('#btnLogin');
-const $checkEmailError = QUERY_SELECTOR('#checkEmailError');
-const $checkPasswordError = QUERY_SELECTOR('#checkPasswordError');
+const ENTER_KEY = 'Enter';
```

Comment:
요건 사소한 부분인데요!
이벤트의 key같은 경우에는 추가로 사용될 가능성이 있다보니까 처음부터 요렇게 만들어줘도 괜찮을 것 같아요!

그리고 `src/constants/event.js` 요런 경로에 파일을 만들어서 상수를 만들어줘도 괜찮을 것 같네요!
```suggestion
const KEY = Object.freeze({
  ENTER: 'Enter',
});
```